frame/scored-pool/src/lib.rs

```diff
@@ -307,7 +319,8 @@ pub mod pallet {
 
 			// can be inserted as last element in pool, since entities with
 			// `None` are always sorted to the end.
```

Comment:
The comment above the `sort_by_key` reads:  
```rust
// Sorts the `Pool` by score in a descending order. Entities which
// have a score of `None` are sorted to the beginning of the vec.
```
so one of the comments must be wrong.